src/main/java/jalau/at18/architects/Wonders/Stage.java

```diff
@@ -1,21 +1,21 @@
-package jalau.at18.architects;
+package jalau.at18.architects.Wonders;
```

Comment:
Please use all lower case for package names, this helps to not confuse them with classes whose names start with a capital letter. `wonders`

Folder name will need to be renamed as well